enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/ChartTypeSelectorList/ChartTypeDropdown.tsx

```diff
@@ -12,7 +12,22 @@ import { useSensibleVisualizations } from "../../hooks/use-sensible-visualizatio
 import ToolbarButtonS from "../../styles/ToolbarButton.module.css";
 import { ToolbarButton } from "../util/ToolbarButton";
 
-export const ChartTypeDropdown = (menuProps: MenuProps) => {
+/**
+ * @category InteractiveQuestion
+ */
+export type InteractiveQuestionChartTypeDropdownProps = MenuProps;
```

Comment:
It's set to `StackProps` on the merged Pr https://github.com/metabase/metabase/pull/56366/files#diff-d0530413cfc45374892b127ac3444a2d47ad9f1ac2e39958bcf84aa33fc80bd9R11. I don't know where `MenuProps` came from

Reply:
It's MenuProps there

![image](https://github.com/user-attachments/assets/a0ad7d0f-144a-4519-bd3e-c2ced8feba9b)